src/components/typography/text/text.mod.css

```diff
@@ -118,3 +131,43 @@ small {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+h1,
+h2,
+h3 {
+  &.headline {
+    color: var(--font-color);
+    margin: 0;
+    font-weight: 300;
+  }
+}
+
+h4.subheadline,
+h5.subheadline {
+  margin: 0;
+  color: var(--font-color);
+
+  &.bold {
+    font-weight: bold;
+  }
+
+  &.primary {
+    color: var(--color-green);
+  }
+
+  &.information {
+    color: var(--color-blue);
+  }
+
+  &.secondary {
+    color: var(--color-gray-60);
+  }
+
+  &.positive {
+    color: var(--color-green-25);
+  }
+
+  &.negative {
+    color: var(--color-red);
+  }
```

Comment:
Could we define separate top-level classes instead? Then we might not need the `postcss-nesting` plugin as well.

So something like

```css
.bold {
  font-weight bold;
}

.positive {
   color: var(--color-green-25);
 }

.subheadline {
   margin: 0;
   color: var(--font-color);
 }
```

We are using the same "modifier" values from what I can tell, so this should reduce the amount of CSS as well.

Reply:
done, I removed `postcss-nesting` and updated the PR to simplify the CSS. VRT still passing. I love VRT.